src/com/company/entities/Order.java

```diff
@@ -0,0 +1,152 @@
+package com.company.entities;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.UUID;
+
+public class Order implements Comparable<Order> {
+    private UUID id;
+    private Restaurant restaurant;
+    private User user;
+    private Courier courier = null;
+    private Adress adress = null;
+    private ArrayList<Product> products = new ArrayList<Product>();
```

Comment:
redundant initialization, you do it in the constructor

Reply:
fixed